libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -457,13 +462,27 @@ export class TurnContext {
         });
 
         return this.emit(this._onSendActivities, output, () => {
-            return this.adapter.sendActivities(this, output)
-                .then((responses: ResourceResponse[]) => {
-                    // Set responded flag
-                    if (sentNonTraceActivity) { this.responded = true; }
-
-                    return responses;
+            if (this.activity.deliveryMode === 'buffered') {
```

Comment:
agreement was to check in with 'bufferedReplies'

Reply:
Resolved via 89f23cb0040ea628ad2c2c9d9ef58b8f15605aaa